src/main/java/nextstep/subway/fee/domain/AgePriceStrategy.java

```diff
@@ -0,0 +1,44 @@
+package nextstep.subway.fee.domain;
+
+import nextstep.subway.line.domain.Line;
+import nextstep.subway.line.domain.Section;
+import nextstep.subway.utils.SectionEdge;
+
+import java.util.List;
+
+import static nextstep.subway.fee.domain.PricePolicy.BASIC_PRICE;
+
+public class AgePriceStrategy {
+
+    private static PricePolicy pricePolicy = new PricePolicy();
+
+    private static int price;
+
+    public int value() {
+        return price;
+    }
+
+    public static int calculatePrice(Integer age) {
+        return getStandardPriceByAge(age);
+    }
+
+    private static int getStandardPriceByAge(int age) {
+        if (pricePolicy.isChildren(age)) {
+            return (int) ((BASIC_PRICE - 350) * 0.5);
+        }
+        if (pricePolicy.isTeenager(age)) {
+            return (int) ((BASIC_PRICE - 350) * 0.2);
+        }
+        return 0;
+    }
```

Comment:
변하는 부분 : `0.5`, `0.2`
변하지 않는 부분: `BASIC_PRICE`, `350`

변하지는 부분과 변하지 않는 부분을 고려해보았을때,
`Enum`을 활용해보면 어떨까 생각이되어요 :thinking: 

```
public enum AgeExample {
    CHILD(0,4,BigDecimal.valueOf(0.1)),
    TEEN(5,10,BigDecimal.valueOf(0.2));

```


```suggestion
    private static int getStandardPriceByAge(AgeExample age) {
      
         return age.calculate((BASIC_PRICE - 350));
    }
```

이런식의 패턴이 된다면, 연령이 추가되어도 `AgeExample`만 변경하면 되겠네요!

Reply:
`BASIC_PRICE` 매직넘버 상수화 해주신것처럼
`350`에 대한부분도 의미있는 이름으로 상수화 해보는것도 좋을것 같네요!

Reply:
이미 알고 계실수도 있지만,
[Java Enum 활용기](https://techblog.woowahan.com/2527/) 공유합니다!